### PR TITLE
Update Miniforge blog post

### LINKED ISSUE #68: installing mambaforge: tell people to scroll down the license agreement
In the post about mambaforge, some people are confused by the license message looking different than in the screenshot. Tell them to scroll down.

### LINKED ISSUE #73: Update mamba blog post to recommend miniforge instead of mambaforge
`mambaforge` is discontinued and it is integrated into `miniforge` now (check message below from miniforge Github repository):

<details>
  <summary>Click to see message</summary>
Mambaforge (Discouraged as of September 2023)
With the [release](https://github.com/conda-forge/miniforge/releases/tag/23.3.1-0) of Miniforge3-23.3.1-0, that incorporated the changes in https://github.com/conda-forge/miniforge/pull/277, the packages and configuration of Mambaforge and Miniforge3 are now identical. The only difference between the two is the name of the installer and, subsequently, the default installation directory.

Given its wide usage, there are no plans to deprecate Mambaforge. If at some point we decide to deprecate Mambaforge, it will be appropriately announced and communicated with sufficient time in advance.

As of September 2023, the new usage of Mambaforge is thus discouraged. Bug reports specific to Mambaforge will be closed as won't fix.

</details>

The link should still be this one: https://github.com/conda-forge/miniforge#miniforge3

But we should point to download and install `miniforge` instead of `mambaforge`. The rest can remain the same, `mamba` should work from `miniforge`.

### PR DESCRIPTION
**Pull request by marabuuu**
Hi @stefanhahmann,

I updated the Miniforge blogpost again and it would be super cool if you could have a look at it. 
I had a chat with Robert and there I found out that we don't need to write `mamba ...` anymore, but can simply use `conda ...`. 

And now, I was trying to find out if I can also remove the `-c conda-forge` from all commands and it seems to be ok, because according to the [conda-forge documentation](https://conda-forge.org/docs/user/introduction/), conda-forge is the default channel of Miniforge.

I feel that these two changes make the installation instructions a bit easier to follow for python beginners, but I am always happy about your input and let me know if you want to have anything changed.

And I just saw that two Issues are still open regarding this blogpost that you Stefan and also Till already addressed, so I will add them here so they are closed when we merge this:
closes #73 
closes #68 

Best,
Mara

### CODE REVIEW

**Review by stefanhahmann — CHANGES_REQUESTED** [review]
Hi @marabuuu,

when I saw your PR, I thought that it would not be a good idea to exchange `mamba` with `conda`, since `mamba` used to be much more efficient than `conda` in solving environments.
However, this does not seem to be the case anymore, at least not with miniforge3, and I was not aware of this.
I just made a test on an independent machine where I freshly installed miniforge3.

I tested 

`conda create -n speed-test-conda devbio-napari python=3.9 pyqt`

vs

`mamba create -n speed-test-mamba devbio-napari python=3.9 pyqt -c conda-forge`

and the environments were created in about the same time (~5 minutes on the machine+network where I tested).

Thus, I agree, since conda may be more well known, it is good idea to replace `mamba` with `conda` in the blog post.

However, I think with your changes, there are still some references to Mamba, which may make it confusing for readers. Some places, where I spotted this:

- One could think about renaming the link of the blog post by renaming the folder in github (and replace mambaforge by miniforge in it): https://biapol.github.io/blog/mara_lampert/getting_started_with_mambaforge_and_python
  - In this case the link to this blog post in this older blog post would need to be replaced: https://biapol.github.io/blog/johannes_mueller/anaconda_getting_started/Readme.html
- The section "Using Mamba" could be renamed to "Using Conda". There are some more mention(s) for "Mamba" in this section, which could be replaced.
  - Please do also check other sections of the blog post for mentions of "Mamba". To avoid confusion I would replace them all by "Conda"

**Comment by stefanhahmann** [comment]
> I updated the Miniforge blogpost again and it would be super cool if you could have a look at it.

Hi @marabuuu I do not see any changes newer than July 8th (cf. https://github.com/BiAPoL/blog/pull/79/commits). Can you check, if you committed and pushed your latest updates? 

You could also consider using `Commit suggestion` and/or `Resolve conversation` / `Reply` to indicate on which of the comments you have worked.

grafik

File: docs/mara_lampert/getting_started_with_mambaforge_and_python/readme.md [hunk 1]
```diff
@@ -3,10 +3,12 @@
...
 
+updated by [Mara Lampert](../readme), July 8th, 2024
+
 `(*)` After the release of Miniforge 23.3.1 in August 2023, Miniforge and Mambaforge are essentially identical. The only difference is the name of the installer and subsequently the default installation path.
 
 ## Introduction to Python and Miniforge 
-This blog post explains what Python and [Mamba](https://mamba.readthedocs.io/en/latest/installation.html) / Miniforge is, and how you can download and setup it on your computer. We will also go through some steps how to get started with Bio-image Analysis. 
+This blog post explains what Python and [Mamba](https://mamba.readthedocs.io/en/latest/) / Miniforge is, and how you can download and setup it on your computer. We will also go through some steps how to get started with Bio-image Analysis. 
 
 > **_Note:_** This is an update of a [previous Blogpost](https://biapol.github.io/blog/johannes_mueller/anaconda_getting_started/) written by Johannes.
```
[the remark this answers is not in the document]
Comment (marabuuu) [reply docs/mara_lampert/getting_started_with_mambaforge_and_python/readme.md:13]: true, thanks for spotting this!

File: docs/mara_lampert/getting_started_with_mambaforge_and_python/readme.md [hunk 2]
```diff
@@ -3,10 +3,12 @@
 
 updated by [Stefan Hahmann](../../stefan_hahmann/readme), December 18th, 2023
 
+updated by [Mara Lampert](../readme), July 8th, 2024
+
 `(*)` After the release of Miniforge 23.3.1 in August 2023, Miniforge and Mambaforge are essentially identical. The only difference is the name of the installer and subsequently the default installation path.
 
 ## Introduction to Python and Miniforge 
-This blog post explains what Python and [Mamba](https://mamba.readthedocs.io/en/latest/installation.html) / Miniforge is, and how you can download and setup it on your computer. We will also go through some steps how to get started with Bio-image Analysis. 
+This blog post explains what Python and [Mamba](https://mamba.readthedocs.io/en/latest/) / Miniforge is, and how you can download and setup it on your computer. We will also go through some steps how to get started with Bio-image Analysis.
```
[the remark this answers is not in the document]
Comment (marabuuu) [reply docs/mara_lampert/getting_started_with_mambaforge_and_python/readme.md:11]: Makes sense, I will link Conda through https://docs.readthedocs.io/en/stable/guides/conda.html instead

**Comment by marabuuu** [comment]
Hi @stefanhahmann,

Thank you for your input and changes! Very cool and helpful. I worked most of you recommendations in, the only thing where I have doubts is changing the link of the blog post:

> One could think about renaming the link of the blog post by renaming the folder in github (and replace mambaforge by miniforge in it): https://biapol.github.io/blog/mara_lampert/getting_started_with_mambaforge_and_python

Because if we change the link then we would need to update all the resources where we mentioned this blogpost and I think this installation blogpost was mentioned in several different resources, not only my own, but also for example Roberts and I would not have the overview in which of these resources I need to update the link.

Is there a way to update a link so that when someone clicks the old link, it automatically opens the new link instead?

Best,
Mara

**Comment by stefanhahmann** [comment]
> Hi @stefanhahmann,
> 
> Thank you for your input and changes! Very cool and helpful. I worked most of you recommendations in, the only thing where I have doubts is changing the link of the blog post: 
> Because if we change the link then we would need to update all the resources where we mentioned this blogpost and I think this installation blogpost was mentioned in several different resources, not only my own, but also for example Roberts and I would not have the overview in which of these resources I need to update the link.

I agree. I had similar thoughts about this. It could even be that externals that we do not even know referenced this blog post and those links would break as well, which would be a pitty.

> Is there a way to update a link so that when someone clicks the old link, it automatically opens the new link instead?

I am not aware of options for link forwarding in the github blog system.

Perhaps, the cleanest solution would have been to actually write new blog post (like we did, to document the changes from anaconda to mambaforge) and reference the new blog post from the current one (like the mambaforge blog post was referenced from the anaconda blog post).

I leave it up to you to decide, if this would be worth the effort.

Best,
Stefan.

**Review by stefanhahmann — APPROVED** [review]

**Pull request closed without merging**